api/middleware/applicationMiddleware.js

```diff
@@ -187,7 +187,7 @@ const sendData = (req, res, next) => {
   }
   // we are going to leave this middleware in this format due to future changes to the database in the near future. updated 7/27/22.
   axios
-    .post(`${process.env.DS_API_URL}/create/${req.role}`, req.application)
+    .post(`${baseURL}/create/${req.role}`, req.application)
```

Comment:
This is a good change that simplifies our requests to the DS API, making them more readable!

Reply:
I believe I resolved the merge conflicts, if there is still an issue please let me know. I also removed the authorization header from the dsConfig file.

Reply:
Excellent, thank you @Nicholas-R-Payne!